service/lotusclient/lotusclient.go

```diff
@@ -92,6 +93,33 @@ func (c *Client) HealthCheck() error {
 	return err
 }
 
+// CurrentSealingSectors returns the total number of sectors considered to be in sealing.
+func (c *Client) CurrentSealingSectors() (int, error) {
+	// these are the sector states mapping to sstProving
+	// https://github.com/filecoin-project/lotus/blob/v1.10.0/extern/storage-sealing/sector_state.go#L109
+	// which are the only states considered not in sealing
+	// https://github.com/filecoin-project/lotus/blob/v1.10.0/extern/storage-sealing/stats.go#L65
+	// hardcode here to avoid importing tons of dependencies.
+	notSealingStates := []string{"Proving", "Removed", "Removing",
```

Comment:
Any reason why defining sealing sectors by the negative?
I guess if they add a new state in the future it would somewhat be the same problem trying to recognize things by direct status or "not in" status, so no strong opinions. I think this is fine.

Reply:
By treating any new states as "sealing" (maybe incorrectly) we throttle bidbot more aggressively to avoid failing to import data for a winning bid.

Reply:
LGTM